.github/workflows/standard.yml

```diff
@@ -1,185 +1,202 @@
-    needs: validate
-
-    strategy:
-      fail-fast: false
-
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-
-        python_version:
-          - "3.12"
-          - "3.11"
-          - "3.10"
-          # - "3.9"  # Not supported
-          # - "3.8"  # Not supported
-
-    name: Create Package
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_create_python_package.yaml@CI-v0.17.0
-    with:
-      operating_system: ${{ matrix.os }}
-      python_version: ${{ matrix.python_version }}
-
-  # ----------------------------------------------------------------------
-  validate_package:
-    needs: create_package
-
-    strategy:
-      fail-fast: false
-
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-
-        python_version:
-          - "3.12"
-          - "3.11"
-          - "3.10"
-          # - "3.9"  # Not supported
-          # - "3.8"  # Not supported
-
-    name: Validate Package
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_validate_python_package.yaml@CI-v0.17.0
-    with:
-      operating_system: ${{ matrix.os }}
-      python_version: ${{ matrix.python_version }}
-      validation_command: python -c "from RepoAuditor import __version__; print(__version__)"
-
-  # ----------------------------------------------------------------------
-  create_binary:
-    needs: validate
-
-    strategy:
-      fail-fast: false
-
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-
-        python_version:
-          - "3.11"
-
-    name: Create Binary
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_create_python_binary.yaml@CI-v0.17.0
-    with:
-      operating_system: ${{ matrix.os }}
-      python_version: ${{ matrix.python_version }}
-
-  # ----------------------------------------------------------------------
-  validate_binary:
-    needs: create_binary
-
-    strategy:
-      fail-fast: false
-
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-
-        python_version:
-          - "3.11"
-
-    name: Validate Binary
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_validate_python_binary.yaml@CI-v0.17.0
-    with:
-      operating_system: ${{ matrix.os }}
-      python_version: ${{ matrix.python_version }}
-      validation_command: RepoAuditor --version
-
-
-
-  # ----------------------------------------------------------------------
-  publish:
-    needs:
-      - validate_package
-      - validate_binary
-
-
-    name: Publish
-
-    uses: davidbrownell/dbrownell_DevTools/.github/workflows/callable_publish_python.yaml@CI-v0.17.0
-    with:
-      release_sources_configuration_filename: .github/release_sources.yaml
-    secrets:
-      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+name: "Standard"
```

Comment:
Just making sure removing the copyright from the top of the file was intentional